docs/adr/AsynkronKoeForBrevBestillinger.md

```diff
@@ -0,0 +1,60 @@
+# Asynkron kø for brevbestillinger
+
+* Status: [draft]
+* Deciders: Håkon Heggholmen
+
+Technical Story: [Lage asynkron kø for brevbestillinger]
+
+## Context and Problem Statement
+
+I dag har vi kun ett synkront løp for bestilling av brev. Det å rendre en PDF med LaTeX tar flere sekunder, og vi har 
+begrenset med ledig synkron kapasitet til en hver tid. For å komme rundt dette problemet i dag må konsumenter hensynta
+våres ledige kapasitet. Konsumenter må også ta stilling til tekniske feil som team pensjonsbrev er ansvarlige for å løse
+, og ha mekanismer for å forsøke på nytt i disse tilfellene. Det koster også penger å alltid ha tilgjengelige instanser 
+kjørende for å kunne håndtere store volum synkrone bestillinger.
+
+## Decision Drivers
+* [Ekstra kompleksitet i løsningene til konsumenter]
+* [Forsinkelser i bestillingsløpet til konsumenter]
+* [Uavklart ansvarsforhold ved tekniske feilmeldinger fra brevløsningen]
+* [Kostnader ved ubrukt reservert CPU kapasitet på NAIS]
+
+## Decision Outcome
+Vi lager en kafka kø. Meldings-størrelsen kan økes, og ved test virker det som om meldingene 
+er godt innenfor grensene. Kafka er godt støttet av NAIS og utviklere har bedre kjennskap til Kafka enn Redis.
+
+### Positive Consequences <!-- optional -->
+
+* Vi skjuler en del tekniske feil som kun vi skal agere på fra konsumenten.
+* Vi samler kompleksitet fra for retry logikk og håndtering av våres tekniske feil ett sted.
+* Vi kan skalere fort basert på antall meldinger i køen.
+* Vi kan spare penger ved å redusere alltid reservert kapasitet på NAIS.
+
+### Negative consequences <!-- optional -->
+
+* Vi får driftsansvar for køen.
+* Vi får mer kompleksitet i kodebasen våres.
+* Vi må integrere tilbake til applikasjonen som kaller oss.
+* 
+## Considered Options
+
+* [option 1] Lage en kafka kø mellom brevbaker og pdf-bygger
```

Comment:
* [option 0] Sette opp rate-limiting av konsumenter, og overlate ansvaret til dem for å oppfylle dette.